landlab/data_record/data_record.py

```diff
@@ -14,40 +15,70 @@ class DataRecord(object):
     Dataset by providing additional attributes and functions, including the
     ability to aggregate values on Landlab grid elements.
 
-    Data variables can vary along one or both of the following dimensions:
+    DataRecord uses the concept of an "item", a physical thing that is located
+    on the grid and has some properties, stored as variables. DataRecord was
+    built to keep track of variables associated with a Landlab grid across
+    items, through time, or both.
+
+    Thus data variables can vary along one or both of the following dimensions:
         - time (model time)
         - item_id: variables can characterize a set of items (each identified
             by an individual id) that reside on the grid.
 
+    If an item or set of items is defined, each item must be defined by the
+    grid element and the element id at which it resides, e.g.:
+
+        grid_element = 'node'
+        element_id = 9.
+
+    When items are defined, each item is given a unique id and the underlying
+    Dataset uses a dimension "item_id".
+
```

Comment:
I've added some text (here and elsewhere) trying to to provide some more background on DataRecord. Let me know what you think.

Reply:
@margauxmouchene thanks for your additional comment on this point. I've added half a sentence based on what you wrote.